doc/yjit/yjit.md

```diff
@@ -94,12 +93,21 @@ make -j install
 or
 
 ```sh
-# Configure in dev (debug) mode for development, build and install
+# Configure in lower-performance dev (debug) mode for development, build and install
 ./autogen.sh
 ./configure --enable-yjit=dev --prefix=$HOME/.rubies/ruby-yjit --disable-install-doc
 make -j install
 ```
 
+Dev mode includes extended YJIT statistics, but can be slow. For only statistics you can configure in stats mode:
```

Comment:
This sounds like `dev` shows more statistics than `stats`, but I believe they both support the same set of statistics. Both of them are slower than release build for YARV insn counters. `dev` is still slower than `stats` due to `RUBY_DEBUG` enabling all interpreter assertions though.

Reply:
Yes, sorry. Both have exactly the same stats. I'll amend to make that clearer.